env/custom.sh

```diff
@@ -1,2 +1,3 @@
 #export CBIOPORTAL_URL="http://localhost:8080"
 #export GENOME_NEXUS_URL="https://www.genomenexus.org"
+export BACKEND="onursumer:fix-cases-without-panel-data"
```

Comment:
you should build against this branch here. it has a fix that hasn't yet been merged to release-5.0.0